io.github.yairm210.unciv.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: io.github.yairm210.unciv
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk8]
+command: unciv-wrapper
+finish-args:
+  - --socket=x11
+  - --device=dri
```

Comment:
Not sure this is really used, but in theory, java can use 2D accelleration trough OpenGL.

Reply:
I'm pretty sure you need it.